pkgs/by-name/cl/clash-verge-rev/package.nix

```diff
@@ -54,6 +49,34 @@ let
       ;
   };
 
+  src-linux = {
+    src = fetchFromGitHub {
+      owner = "clash-verge-rev";
+      repo = "clash-verge-rev";
+      tag = "v${version}";
+      hash = "sha256-HBWvk6bX0GjU/yvUejYgTQM8/IP5dYVrf30wNzgWv0s=";
+    };
+  };
+  sources = {
+    aarch64-darwin = {
+      src = fetchurl {
+        url = "https://github.com/clash-verge-rev/clash-verge-rev/releases/download/v${version}/Clash.Verge_${version}_aarch64.dmg";
+        sha256 = "sha256-V7COnyVLsaNcHHr8iK9X2a3ZapUrFavQx9ycYrEojmY=";
+      };
+    };
+    x86_64-darwin = {
+      src = fetchurl {
+        url = "https://github.com/clash-verge-rev/clash-verge-rev/releases/download/v${version}/Clash.Verge_${version}_x64.dmg";
+        sha256 = "sha256-Gk6l7yuja9/6wI1KODHyYP2VG7YoUEz9wVKNry/kPJE";
+      };
+    };
+    x86_64-linux = src-linux;
+    aarch64-linux = src-linux;
+    i686-linux = src-linux;
+  };
+  source = sources.${stdenv.hostPlatform.system};
```

Comment:
```suggestion
  };
  source = if stdenv.isDarwin then src-darwin.${stdenv.hostPlatform.system} else src-linux;
```

Reply:
Let's not specify the architectures on Linux in case others try to build it on their computer.